admin/links/class-link-table-accessible.php

```diff
@@ -8,28 +8,63 @@
  */
 class WPSEO_Link_Table_Accessible {
```

Comment:
For some reason I think this is the time to introduce an abstract class that contains the duplicate functionality of `WPSEO_Link_Table_Accessible` and  `WPSEO_Meta_Table_Accessible `

Reply:
We cannot, as these have `static public` functions which cannot be put in an abstract class.
As we are going to move towards using an ORM, no other classes like this will be created in the future.